src/PrestaShopBundle/Service/Form/MultistoreCheckboxEnabler.php

```diff
@@ -188,7 +197,6 @@ private function addCheckbox(FormInterface $form, string $relatedFieldName, bool
         $form->add($fieldName, CheckboxType::class, [
             'required' => false,
             'data' => $isOverriddenInCurrentContext,
-            'multistore_configuration_key' => $configurationKey,
```

Comment:
@matthieu-rolland we removed this field because we realized it was actually never used So its presence was confusing